Quantum-Control-Applications/NV-Centers/Cryogenic nanophotonic cavity/README.md

```diff
@@ -0,0 +1,43 @@
+# Single Yb center in a cyrogenic nanophotonic cavity
+
+## Basic Files
+These files showcase various experiments that can be done on a Yb center in cryogenic nanophotonic cavity with a SNSPD 
+and an AOM which is controlled via a digital channel.
```

Comment:
So here, as opposed to the use-case, the tone driving the AOM is not generated by the OPX?